lstmcpipe/utils.py

```diff
@@ -80,7 +186,194 @@ def dump_lstchain_std_config(filename='lstchain_config.json', allsky=False, over
     print(f"Updating std lstchain config {extra_msg} with")
     diff = DeepDiff(std_cfg, cfg)
     pprint(diff)
-    
+
     with open(filename, 'w') as file:
         json.dump(cfg, file, indent=4)
     print(f"\nModified lstchain config dumped in {filename}. Check full config thoroughly.")
+
+
+def run_command(*args):
+    """
+    Runs the command passed through args, as a subprocess.run() call.
+
+    Based on:
+    https://github.com/cta-observatory/cta-lstchain/blob/master/lstchain/scripts/tests/test_lstchain_scripts.py#L43
+
+    Parameters
+    ----------
+    args: str or iter
+        `Shell` is forced to True, thus a single string (shell cmd) is expected.
+
+    Returns
+    -------
+    (subprocess.Popen.stdout).strip('\n')
+    """
+    cmd = sp.Popen(args, shell=True, stdout=sp.PIPE, stderr=sp.STDOUT, encoding='utf-8')
```

Comment:
> subprocess.run executes a command and waits for it to finish 
`run` will run interactively all the batched commands. It will stop on the first batch command (`r0_to_dl1`) until it is finished, and then it will just go to the next next (another `r0_to_dl1`) and waits it is finished.... Meaning that the whole   `lstmcpipe` program will run interactively !

Reply:
Same reason why we were using before `os.popen` instead of `os.system`

Reply:
I don't think that's true at all !
`run_command` is used to submit a slurm job, the command is just sending that job, nothing more.

Reply:
Right,

will change it whenever I have some time 

```
a = sp.run(['sbatch', '--parsable', '--wrap=sleep 10"'], stdout=sp.PIPE, stderr=sp.STDOUT, encoding='utf-8')

In [21]: a.stdout
Out[21]: '155653\n'

In [22]: a.returncode
Out[22]: 0
```